### PR TITLE
Dump events from e2e tests

### PR DESCRIPTION
**Pull request by kargakis**
Signed-off-by: Michail Kargakis <mkargaki@redhat.com>

### CODE REVIEW

**Comment by kargakis** [comment]
[test]

File: hack/lib/cleanup.sh [hunk 1]
```diff
@@ -152,4 +152,18 @@ function os::cleanup::tmpdir() {
...
+# os::cleanup::dump_events dumps all the events from a cluster to a file.
+#
+# Globals:
+#  ARTIFACT_DIR
+# Arguments:
+#  None
+# Returns:
+#  None
+function os::cleanup::dump_events() {
+	os::log::info "Dumping cluster events to ${ARTIFACT_DIR}/events.txt"
```
Comment (stevekuznetsov) [inline hack/lib/cleanup.sh]: make this `debug`

File: hack/lib/cleanup.sh [hunk 2]
```diff
@@ -152,4 +152,18 @@ function os::cleanup::tmpdir() {
...
+	oc get events --all-namespaces > "${ARTIFACT_DIR}/events.txt"
```
Comment (stevekuznetsov) [inline hack/lib/cleanup.sh]: does this require admin? could use `--config="${ADMIN_KUBECONFIG}"` if that's set

File: hack/lib/cleanup.sh (hunk 1 above)
Comment (kargakis) [inline hack/lib/cleanup.sh]: Not sure if it's worth moving at "debug". More often than not when debugging e2e I would prefer to see by default that a file with all the events has been placed somewhere. It's the same level as the dump we get from etcd, too.

Comment (stevekuznetsov) [inline hack/lib/cleanup.sh]: Fair enough. We still get `debug` level in the `scripts.log` artifact but OK.

**Comment by kargakis** [comment]
https://github.com/openshift/origin/issues/13966

File: hack/lib/cleanup.sh (hunk 2 above)
Comment (kargakis) [inline hack/lib/cleanup.sh]: done

**Comment by kargakis** [comment]
https://github.com/openshift/origin/issues/10738 [test]

**Comment by kargakis** [comment]
[merge]

**Pull request merged**